lib/models/fire.dart

```diff
@@ -102,7 +102,7 @@ class Fire {
     );
   }
 
-  static FireStatus _statusFromJson(String status) {
```

Comment:
why did you change `_statusFromJson` to `statueFromJson` ?

Reply:
Solved in `efbbc7e`.